cli/cmd/testdata/inject_emojivoto_deployment_empty_version_config.golden.yml

```diff
@@ -0,0 +1,150 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: web
+  namespace: emojivoto
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: web-svc
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        linkerd.io/created-by: linkerd/cli dev-undefined
+        linkerd.io/identity-mode: default
+        linkerd.io/proxy-version: dev-undefined
```

Comment:
If neither `global.version` or `proxy.proxy_version` are set in the config map, falls back to `version.Version`.